packages/thirdweb/src/wallets/create-wallet.ts

```diff
@@ -118,6 +117,7 @@ export function createWallet<const ID extends WalletId>(
             WCSupportedWalletIds | InjectedSupportedWalletIds
           >,
         ) => {
+          const { injectedProvider } = await import("./injected/mipdStore.js");
```

Comment:
is this change still necessary?

Reply:
not anymore, but it doesn't hurt right? any downside?

Reply:
doesn't hurt per se no